src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1819,6 +1769,16 @@ public bool ShouldDisplayPreview
         }
 
         protected bool ShouldDisplayPreviewCore { get; set; }
+        
+        public event Action<NodeModel, IEnumerable<IRenderPackage>> RenderPackagesUpdated;
```

Comment:
Can we move this event to an interface? This might make it easier to consume non-`NodeModel` geometry data sources - this would make `Watch3DViewModel` far more reusable.